### DISCUSSION TITLE
Hide Side Panel Button Does Not Work. (macOS, arm64)

### DISCUSSION THREAD

**Discussion by BingoKingo** [Alex313031/Thorium-Special#27] [Q&A] [answered]
I change the Option in `chrome://flags`, but it does not work.
How to fix it?

By the way, how to hide the experimental feature button 
<img width="31" alt="image" src="https://user-images.githubusercontent.com/24961267/195120036-40ed749c-6a93-4d59-8f0c-4efa8f64e00a.png">
and the tab search button
<img width="43" alt="image" src="https://user-images.githubusercontent.com/24961267/195120155-4f18ac3b-5ebb-41ee-be1b-f61795b42ad9.png">
? Will it be supported?

And I like the default button style 
<img width="92" alt="image" src="https://user-images.githubusercontent.com/24961267/195124710-1444a659-97ec-4a1f-bf40-55af91f3803a.png">
instead of modified one,
<img width="92" alt="image" src="https://user-images.githubusercontent.com/24961267/195124948-93e7d3c8-7d29-476e-98de-98469e87bcdd.png">
how to set to default?

**Comment by gz83** [comment]
> I change the Option in `chrome://flags`, but it does not work. How to fix it?
> 
> By the way, how to hide the experimental feature button? Will it be supported?

@BingoKingo Hello, please find the **Hide Side Panel Button** in the chrome://flags page and set it to **Enabled**, then restart the browser to hide the sidebar button

**Reply by BingoKingo** [reply]
Hi, set to enabled doesn't work. The appearance is not changed.
<img width="143" alt="image" src="https://user-images.githubusercontent.com/24961267/195119842-f1fdeb74-f612-4d73-9f4b-ce99bfb417c4.png">

**Reply by gz83** [reply]
> Hi, set to enabled doesn't work. The appearance is not changed.

@BingoKingo Can you send a screenshot of the location of the side panel button you want to hide?

**Reply by BingoKingo** [reply]
<img width="58" alt="image" src="https://user-images.githubusercontent.com/24961267/195125648-a60ea12b-dcfb-48b1-8535-664b89eb2a1c.png">
This one called **Side Panel Button**

**Reply by gz83** [reply]
> <img alt="image" width="58" src="https://user-images.githubusercontent.com/24961267/195125648-a60ea12b-dcfb-48b1-8535-664b89eb2a1c.png">
> 
> This one called **Side Panel Button**

@BingoKingo Are you using the latest version?

**Reply by BingoKingo** [reply]
This Version: <https://github.com/Alex313031/Thorium-Special/releases/tag/M107.0.5279.0_1>

**Reply by gz83** [reply]
> This Version: https://github.com/Alex313031/Thorium-Special/releases/tag/M107.0.5279.0_1

@BingoKingo Can you provide a screenshot of the **chrome://version/** page?

**Reply by BingoKingo** [reply]
<img width="846" alt="image" src="https://user-images.githubusercontent.com/24961267/195135596-f349846f-708d-4c7a-ae60-13c7af6220d5.png">

**Reply by gz83** [reply]
> <img alt="image" width="846" src="https://user-images.githubusercontent.com/24961267/195135596-f349846f-708d-4c7a-ae60-13c7af6220d5.png">

@BingoKingo Please uninstall the browser and reinstall it again to see if the side panel button can be hidden.

Be careful not to clear your browser's data when uninstalling.

**Reply by BingoKingo** [reply]
@gz83 I tried uninstall and reinstall before. So what is the problem?

**Reply by gz83** [reply]
> @gz83 I tried uninstall and reinstall before. So what is the problem?

@BingoKingo The problem is weird, I will report this to @Alex313031 @midzer

**Comment by Alex313031** [comment]
@BingoKingo Does the hide side panel still not work? ALso you can get rid of the chrome labs button in chrome://flags

**Reply by BingoKingo** [reply, marked as answer]
Solved on newer version, thanks.
